shared/js/smart-screen/scrollable.js

```diff
@@ -462,6 +464,10 @@
       this._setNodePosition(newIdx);
 
       this.spatialNavigator.add(itemElem);
+      if (this.listElem.classList.contains('reversed')) {
```

Comment:
Put a "XXX" before if clause and describe why we have to do realign and shift for just `reversed` case